docs/section23/Rule23-16.md

```diff
@@ -5,42 +5,43 @@
 **Mandatory Rule:** True  
 **Rule ID:** 23-16  
 **Rule Description:** Systems 5 - 8, the baseline system shall be modeled with preheat coils controlled to a fixed set point 20F less than the design room heating temperature setpoint.  
-**Rule Assertion:** B-RMI = expected value  
+**Rule Assertion:** B-RMR = expected value  
 **Appendix G Section:** Section 23 Air-side  
 **90.1 Section Reference:** Section G3.1.3.19 Preheat Coils (Systems 5 through 8)  
 **Data Lookup:** None  
 **Evaluation Context:** Building  
 
 **Applicability Checks:**  
 
-1. B-RMI is modeled with at least one air-side system that is Type-5, 6, 7, 8, 7a, 8a, 5b, 6b, 7b, 8b, 7c.  
+1. B-RMR is modeled with at least one air-side system that is Type-5, 6, 7, 8, 7a, 8a, 5b, 6b, 7b, 8b, 7c.  
```

Comment:
Line 56 expects preheat_b.heating_system_type == "FLUID_LOOP" in order for the True outcome, but shouldn'd preheat be electric resistance for Systems 6 and 8? Recommend removing this condition.

Reply:
the work that I've done on this rule is only related to schema updates, but this makes sense.  Removed both references to water coils:

`( preheat_b.heating_system_type == "FLUID_LOOP" ) AND ( get_component_by_id(preheat_b.hot_water_loop).type == "HEATING" )`

The only check now is related to the preheat coil setpoint temperature: 

`( preheat_b.heating_coil_setpoint == hvac_max_zone_setpoint_dict[hvac_b] - 20F )`